account_analytic_project/i18n/account_analytic_project.pot

```diff
@@ -0,0 +1,51 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+#	* account_analytic_project
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 6.1\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-02-19 10:38+0000\n"
+"PO-Revision-Date: 2015-02-19 10:38+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: account_analytic_project
+#: code:addons/account_analytic_project/model/account_analytic_account.py:64
+#, python-format
+msgid "This method is only valid for a single analytic account"
+msgstr "This method is only valid for a single analytic account"
```

Comment:
The pot file is the mostly unaltered result of exporting the English language from the module. That has all the msgstr's filled. Is this a problem? It is not a big problem to remove them from the few lines for this module, but other modules might have a lot of text, so If not needed, I would be glad to leave them the way they were exported.